src/main/resources/lang/english.lang

```diff
@@ -394,6 +394,9 @@ biomes:
 	soul_sand_valley: soul sand valley
```

Comment:
not sure where to put this so I'm just putting it here lol

I think you still need to add the composter inventory type

Reply:
added, thanks for the remind